securing_apps/topics/oidc/java/fuse7/classic-war.adoc

```diff
@@ -83,8 +84,31 @@ To enable the functionality, add this section to your `/WEB_INF/web.xml` file:
 </context-param>
 ----
 
-That component uses `keycloak.config` or `karaf.etc` java properties to search for a base folder to locate the configuration.
-Then inside one of those folders it searches for a file called `<your_web_context>-keycloak.json`.
+The available resolvers for `keycloak.config.resolver` are the following ones:
+
```

Comment:
I suggest: "The following resolvers are available for `keycloak.config.resolver`: